google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/VerifyAdvertiserIdentity.java

```diff
@@ -0,0 +1,173 @@
+            customerId);
+        System.out.println(
+            "See https://support.google.com/adspolicy/answer/9703665 for details on how and when an"
+                + " account is required to undergo the advertiser identity verification program.");
+        return;
+      }
+
+      IdentityVerificationProgramStatus programStatus =
+          identityVerification.getVerificationProgress().getProgramStatus();
+      switch (programStatus) {
+        case UNSPECIFIED:
+          // Starts an identity verification session.
+          startIdentityVerification(customerId, identityVerificationServiceClient);
+
+          // Calls getIdentityVerification again to retrieve the verification progress after
+          // starting an identity verification session.
+          getIdentityVerification(customerId, identityVerificationServiceClient);
+          break;
+        case PENDING_USER_ACTION:
+          // If there is an identity verification session in progress, there is no need to start
+          // another one by calling startIdentityVerification.
+          System.out.println("There is an advertiser identify verification session in progress.");
+          System.out.printf(
+              "The URL for the verification process is '%s' and it will expire at '%s'%n",
+              identityVerification.getVerificationProgress().getActionUrl(),
+              identityVerification.getVerificationProgress().getInvitationLinkExpirationTime());
+          break;
+        case PENDING_REVIEW:
+          System.out.println("The verification is under review.");
+          break;
+        case SUCCESS:
+          System.out.println("The verification already completed.");
+          break;
+        default:
+          throw new IllegalStateException(
+              "The identity verification has an unexpected status: " + programStatus);
+      }
+    }
+  }
+
+  /** Retrieves the status of the advertiser identity verification process. */
+  // [START verify_advertiser_identity_1]
+  private IdentityVerification getIdentityVerification(
+      long customerId, IdentityVerificationServiceClient identityVerificationServiceClient) {
+    GetIdentityVerificationResponse response =
+        identityVerificationServiceClient.getIdentityVerification(Long.toString(customerId));
+    if (response.getIdentityVerificationCount() == 0) {
+      return null;
+    }
+    IdentityVerification identityVerification = response.getIdentityVerification(0);
+    String deadline =
+        identityVerification
+            .getIdentityVerificationRequirement()
+            .getVerificationCompletionDeadlineTime();
+    IdentityVerificationProgress progress = identityVerification.getVerificationProgress();
+    System.out.printf(
+        "Account %d has a verification completion deadline of '%s' and status '%s' for advertiser"
+            + " identity verification.%n",
+        customerId, deadline, progress.getProgramStatus());
+    return identityVerification;
+  }
+
```

Comment:
Nit: needless empty line

Reply:
For some reason, the Google Java Formatter insists on adding this empty line whenever I try to take it out. I'd prefer to leave it in so we don't have to manually fix it any time we touch this file going forward.

Reply:
🤷🏻